packages/rn-tester/js/components/RNTesterHeader.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import {
+  Text,
+  View,
+  SafeAreaView,
+  Button,
+  Platform,
+  StyleSheet,
+} from 'react-native';
+import * as React from 'react';
+import RNTesterDocumentationURL from './RNTesterDocumentationURL';
+import {RNTesterThemeContext} from './RNTesterTheme';
+
+const HeaderIOS = ({
+  onBack,
+  title,
+  documentationURL,
+}: {
+  onBack?: () => mixed,
+  title: string,
+  documentationURL?: string,
+  ...
```

Comment:
Is this not sealed? Could we remove the `...`?

Reply:
Done!